config/config.yaml

```diff
@@ -1,4 +1,13 @@
 appVersion: 0.1
 appName: PayGoGateway
-config:
-  apiUrl: "https://api.mercadopago.com/"
\ No newline at end of file
+config: ""
+mercadopago:
+  config:
+    apiUrl: "https://api.mercadopago.com/"
+    apiVersion: "v1"
+    apiClientID: "7130376828807209"
```

Comment:
Not is a good practice put config files or credentials on version control, the better way is put a `config.example.yaml` only with the necessary fields.
 
Why?

* If you keep this, new people can't use they credentials or another fields.
* You expose your credentials for anyone (Security).

A better way to store config is in env vars. read this (https://12factor.net/pt_br/config)

Reply:
I agree, my bad